tracing-core/src/field.rs

```diff
@@ -1081,6 +1081,7 @@ mod private {
     pub trait ValidLen<'a>: Borrow<[(&'a Field, Option<&'a (dyn Value + 'a)>)]> {}
 
     impl<'a, const N: usize> ValidLen<'a> for [(&'a Field, Option<&'a (dyn Value + 'a)>); N] {}
+    impl<'a> ValidLen<'a> for &[(&'a Field, Option<&'a (dyn Value + 'a)>)] {}
```

Comment:
Oh, hmm, I just noticed I elided the first lifetime in the slice type. Should I make that `'a` as well, do you think?